src/helpers/time.js

```diff
@@ -57,3 +58,15 @@ export const getMinimalTimeUnitStringForMs = (
 
   return `${highestResolutionValue} ${label}`;
 };
+
+/**
+ * @desc get time in microseconds and return a timestamp string
+ * @param {String} [handshake='']
+ * @return {String}
+ */
```

Comment:
Appreciate this, but realistically this is not needed. TS should fix this eventually